### ISSUE TITLE
missing introduction to the format

### ISSUE THREAD

**Issue by matkoniecz** [public-transport/friendly-public-transport-format#52]
https://github.com/public-transport/friendly-public-transport-format/blob/master/README.md describes the vision, without any examples of the format itself

it links to https://github.com/public-transport/friendly-public-transport-format/blob/master/spec/readme.md that has no introduction or basic examples. It immediately starts from "In order to be able to use this format in both static data and API responses, you may chose wether to reference an item by its id or inline the item as an object." what is a minor, not so important detail.

Maybe it would be better to start from a straightforward example of a simplest possible route?

In my case I am looking for a simple format for representing train/bus/plane routes and I prefer to avoid inventing my own from scratch. But I am also uninterested in looking through entire spec to implement simple things following it.

**Comment by matkoniecz** [comment]
Now I started reading rest of documentation and it is less problematic than expected as it is quite short and uncomplicated.

But I will leave it open as first impression is quite poor (feel free to close is that is not a problem or not worth fixing).

**Comment by matkoniecz** [comment]
I just found https://github.com/public-transport/friendly-public-transport-format/blob/master/examples/valid-simple-journey.json

I think it should be more accessible (I can make a PR, ping me after processing ones that I already created)

**Comment by derhuerst** [comment]
Agree, the first impression is bad. Let's add a really basic example, e.g. the one from https://github.com/public-transport/friendly-public-transport-format/blob/a262d2814217da9ceab7798682c97b7a1e36c49b/examples/valid-simple-journey.json

**Comment by matkoniecz** [comment]
valid-simple-journey.json may be further simplified. departureDelay and arrivalDelay are both optional and something that may be safely dropped from simplest possible example.

I am considering further simplification, I created #55 and #56 to check is it something possible.

**Comment by derhuerst** [comment]
I think the duplicated `price` can be left out from the introduction. `departureDelay`/`arrivalDelay` are IMO quite important, and therefore one of them should be included.
